client.go

```diff
@@ -791,6 +793,7 @@ func (c *Client) loadServerCert(cert string) error {
 	certPool.AddCert(x509Cert)
 
 	c.config.TLSConfig.RootCAs = certPool
+	c.config.TLSConfig.ClientCAs = certPool
```

Comment:
Is there any harm to setting ClientCAs to certPool all the time (e.g. if AutoMTLS is not enabled)?

Reply:
The client will only call this method if AutoMTLS is enabled. The server will [send an empty string](https://github.com/hashicorp/go-plugin/blob/73def4eb51d997424d91b390f8d5bd2bae7b9e5b/server.go#L411) if AutoMTLS is false which will be [checked by the client](https://github.com/hashicorp/go-plugin/blob/73def4eb51d997424d91b390f8d5bd2bae7b9e5b/client.go#L764).

Reply:
Ah, right! Thanks for pointing that out.